Dockerfile

```diff
@@ -14,7 +14,7 @@ ARG FLATBUFFERS_IMAGE_BASE="debian"
 ARG FLATBUFFERS_IMAGE_TAG="bullseye-slim"
 
 ARG FLATBUFFERS_ARCHIVE_BASE_URL="https://api.github.com/repos/google/flatbuffers/tarball"
-ARG FLATBUFFERS_REPO_TAG="master"
+ARG FLATBUFFERS_ARCHIVE_TAG="master"
```

Comment:
The `FLATBUFFERS_REPO_TAG` is not used anywhere. From my reading of the entire code, I believe the correct term is `FLATBUFFERS_ARCHIVE_TAG`.